pkgs/desktops/gnome-3/extensions/switcher/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-switcher";
+  version = "26";
+
+  src = fetchFromGitHub {
+    owner = "daniellandau";
+    repo = "switcher";
+    rev = "b707bdfcfaac9bf6e73e162ab75b871eb920ae41";
+    sha256 = "1hx078w8mzaflwk8kd7s8np4041mbgca86rg7iyb9g8wbrbb0nzv";
+  };
+
+  uuid = "switcher@landau.fi";
+
+  nativeBuildInputs = [ glib ];
```

Comment:
Is this used for anything? The git repo comes with compiled schemas, so there's no need for `glib-compile-schemas` at least.